aqua-examples/ts-oracle/Readme.md

```diff
@@ -106,84 +161,140 @@ data Oracle:
     raw_data: []u64
 
 -- the point_estimate function from the Wasm file wrapped as a service
--- marine aqua artifacts/ts_oracle.wasm
+-- marine aqua ts-oracle/modules/ts-oracle/target/wasm32-wasi/release/ts_oracle.wasm
 service TSOracle("service-id"):
     point_estimate: []u64, u32 -> Oracle
 
 -- the timestamp getter drawing from the Kademlia neighborhood
 -- function collects the neighborhood peers
 -- retrieves and returns the timestamps 
-func ts_getter(node: string) -> []u64:
+-- the peer id gets from the wrapper of our service interface
+-- for handling timeouts from the neighborhood peers, the race pattern is used
```

Comment:
it's the race pattern implementation and should be linked, https://doc.fluence.dev/aqua-book/language/flow/parallel#timeout-and-race-patterns

Reply:
Agree